core/http/client/src/main/java/org/eclipse/rdf4j/http/client/QueueCursor.java

```diff
@@ -29,11 +29,11 @@
 
 	private volatile boolean done;
 
-	private BlockingQueue<E> queue;
+	private final BlockingQueue<E> queue;
 
-	private E afterLast = createAfterLast();
+	private final E afterLast = createAfterLast();
 
-	private volatile Queue<Throwable> exceptions = new LinkedList<Throwable>();
+	private final Queue<Throwable> exceptions = new LinkedList<Throwable>();
```

Comment:
This is technically a performance change, as it will help compiler avoid three volatile reference resolutions per getNextElement call since we were never reassigning this private variable anyway.